neo/Network/RPC/RpcServer.cs

```diff
@@ -90,12 +90,17 @@ private JObject GetInvokeResult(byte[] script)
             return json;
         }
 
-        private static JObject GetRelayResult(RelayResultReason reason)
+        private static JObject GetRelayResult(RelayResultReason reason, UInt256 hash)
         {
             switch (reason)
             {
                 case RelayResultReason.Succeed:
-                    return true;
+                    {
+                        var ret = new JObject();
+                        ret["relayed"] = true;
+                        ret["hash"] = hash.ToString();
```

Comment:
I think we should call this `txid` instead of `hash` to stay consistent. e.g. [sendtoaddress](
https://docs.neo.org/en-us/node/cli/latest-version/api/sendtoaddress.html) and [sendfrom](https://docs.neo.org/en-us/node/cli/latest-version/api/sendfrom.html) also use `txid`

Otherwise looks good

Reply:
It's weird to return two different structures for blocks and transactions.

Reply:
Maybe we can leave blocks as before, but looks weird too

Reply:
Maybe you should use `hash` uniformly.

Reply:
agree